### ISSUE TITLE
The step size of initial mouse movement is too big.

### ISSUE THREAD

**Issue by whileloopa** [neatdecisions/neatmouse#3]
The step size of initial mouse movement is too big while the following movement  is in small step size.

I suggest decrease the initial step size and use fast small steps instead of a big step.

**Comment by Xachman** [comment]
@whileloopa  Are you talking about the initial speed settings?

**Comment by nexurion** [comment]
@Xachman I join the discussion. Please make a smooth increase in the speed of the cursor movement. for example: in the Xpadder program, if you briefly press the d-pad button, which is assigned to the cursor control, the cursor gradually increases the speed and it can be adjusted.

**Comment by ZombiKain** [comment]
@Xachman The main problem of this emulator is that the cursor "accelerates" sharply. That is, when you press the key to move the cursor, it immediately moves at the set speed. There is no smoothness in the first fractions of a second.

And I think this can be fixed, because in another emulator, already for gamepads, Xpadder, if you control the cursor with the gamepad buttons, the cursor accelerates smoothly and you can get to exactly where you need. I'm slowly learning 3D modeling and so far I have to model from the gamepad. It's not so painful to work with him. But I would like to model from the keyboard and do other things.

**Comment by uzayyli** [comment]
Yes, please add an acceleration profile. Maybe just linear acceleration and let us set the parameter `time_to_max_speed` like in xkbset mousekeys.
